### PR TITLE
c2ixys

### PR DESCRIPTION
**Pull request by prakharcode**

### CODE REVIEW

**Comment by helgee** [comment]
This function should be called `celestial_to_intermediate`

File: REQUIRE [hunk 1]
```diff
@@ -1 +1,3 @@
 julia 0.6
+ERFA 0.2.2
```
Comment (helgee) [inline REQUIRE]: `ERFA` should be a test-only dependency, i.e. put it in `test/REQUIRE`

File: src/AstroBase.jl [hunk 2]
```diff
@@ -1,5 +1,13 @@
 module AstroBase
 
+using Rotations
+export c2ixys
 # package code goes here
+function c2ixys(x, y, s)
+    r2 = x^2 + y^2
+    e = r2 > 0.0? atan2(y, x) : 0.0
```
Comment (helgee) [inline src/AstroBase.jl]: Whitespace around `?`

File: src/AstroBase.jl [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+    d = atan(sqrt(r2 / (1.0 - r2)))
+    RotZYZ(e, d, -(e+s))
```
Comment (helgee) [inline src/AstroBase.jl]: Whitespace around `+`

File: src/AstroBase.jl [hunk 4]
```diff
@@ -1,5 +1,27 @@
...
+julia> celestial_to_intermediate(0.2, 0.2, 0.1)
+3×3 RotZYZ{Float64}(0.785398, 0.286757, -0.885398):
+  0.976728   0.0774803  0.2
+ -0.11811    0.972651   0.2
+ -0.179034  -0.218968   0.959166
+```
+"""
+function celestial_to_intermediate(x, y, s)
+    r2 = x^2+y^2
+    e = r2 > 0.0?atan2(y,x):0.0
```
Comment (helgee) [inline src/AstroBase.jl]: My comment was meant to be read "add whitespace around operators". Sorry 🙈

Comment (prakharcode) [reply src/AstroBase.jl]: oh! doing it!

File: src/AstroBase.jl [hunk 5]
```diff
@@ -1,5 +1,27 @@
 module AstroBase
 
+using Rotations
+export celestial_to_intermediate
 # package code goes here
+"""
+    celestial_to_intermediate(x, y, s)
+
+Converts the celestial to intermediate-frame-of-date matrix given the CIP
```
Comment (helgee) [inline src/AstroBase.jl]: s/Converts/Returns/

File: src/AstroBase.jl [hunk 6]
```diff
@@ -1,5 +1,27 @@
...
+  0.976728   0.0774803  0.2
+ -0.11811    0.972651   0.2
+ -0.179034  -0.218968   0.959166
+```
+"""
+function celestial_to_intermediate(x, y, s)
+    r2 = x^2 + y^2
+    e = r2 > 0.0 ? atan2(y,x) : 0.0
+    d = atan(sqrt(r2 / (1.0 - r2)))
+    RotZYZ(e, d, - (e + s))
```
Comment (helgee) [inline src/AstroBase.jl]: Unary operators do not get whitespace, i.e. `-(e + s)`

**Pull request merged**